README.md

```diff
@@ -60,7 +59,7 @@ custom:
     pinVersion: 1.31.1
 ```
 
-In case you want to manage the Lumigo tracer dependency yourself - e.g. you want to use Lerna or Webpack, and can't have this plugin install the Lumigo tracer on your behalf on every deployment - then you can also disable the NPM install process altogether.
+In some cases, such as when working with tools like Lerna or Webpack, you may not want the Serverless plugin to install the Lumigo tracer on your behalf for every deployment. If you encounter this, you can disamle the NPM installation process altogether using the `skipInstallNdoeTracer` configuration setting:
```

Comment:
small typo here - `disamble` instead of `disable`

Reply:
Thanks for the catch! I fixed the typo